sorc/build_all.sh

```diff
@@ -129,7 +133,7 @@ declare -A build_opts
 big_jobs=0
 build_jobs["ufs"]=8
 big_jobs=$((big_jobs+1))
-build_opts["ufs"]="${_wave_opt} ${_verbose_opt} ${_build_ufs_opt} ${_build_debug}"
+build_opts["ufs"]="${_wave_opt} ${_hydro_opt} ${_verbose_opt} ${_build_ufs_opt} ${_build_debug}"
```

Comment:
This requires `build_ufs.sh` to be updated as well to make use of the `-y`.

Reply:
@aerorahul I didn't catch an error with the merge/commit. The updated build_all.sh and other files are not uploaded